### PR TITLE
th-14: Create header component

### PR DESCRIPTION
**Pull request by Some14u**
A simplest ever possible header. 
изображение

The logo consists of two svg images. Logo component accepts optional prop called "layout", you can pass "horizonal" or "vertical". Vertical layout show the brand name under the logo symbol.

Also i made logo in that way so as long as the container has specified width, it will adjust it's size respectively. This way you can make logo of any size by explicitly specifying the width of it's container:
изображение

### CODE REVIEW

File: frontend/src/libs/components/app-logo/app-logo.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+type AppLogoProperties = {
+  layout?: 'horizontal' | 'vertical';
+};
+
+const AppLogo: React.FC<AppLogoProperties> = ({
+  layout = 'horizontal',
+}: AppLogoProperties) => {
+  return (
+    <div className={[styles.container, styles[layout]].join(' ')}>
```
Comment (Druxight) [inline frontend/src/libs/components/app-logo/app-logo.tsx]: use clsx for that

File: frontend/src/libs/components/app/app.tsx [hunk 2]
```diff
@@ -27,9 +27,15 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  const handleSighIn = useCallback(() => {
```
Comment (Druxight) [inline frontend/src/libs/components/app/app.tsx]: handleSignIn function should be inside Header component

File: frontend/src/libs/components/app/app.tsx [hunk 3]
```diff
@@ -27,9 +27,15 @@ const App: React.FC = () => {
...
+    return;
+  }, []);
+
   return (
     <>
-      <img src={reactLogo} className="App-logo" width="30" alt="logo" />
+      <header>
```
Comment (Druxight) [inline frontend/src/libs/components/app/app.tsx]: The header element should be in the header component

**Review by Druxight — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/app/app.tsx (hunk 2 above)
Comment (Some14u) [reply frontend/src/libs/components/app/app.tsx]: Great, 100% agree. It was outside in our thread.js task though.

File: frontend/src/libs/components/app-logo/app-logo.tsx (hunk 1 above)
Comment (Some14u) [reply frontend/src/libs/components/app-logo/app-logo.tsx]: did it

File: frontend/src/libs/components/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
```
Comment (Druxight) [inline frontend/src/libs/components/header/header.tsx]: We have a custom component Link for that (frontend/src/libs/components/link/link.tsx)

**Review by Druxight — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/header/header.tsx (hunk 4 above)
Comment (Some14u) [reply frontend/src/libs/components/header/header.tsx]: i see, let me use it then

**Review by Druxight — APPROVED** [review]

**Review by PavloSukhinin — APPROVED** [review]
Button should be replaced later

**Pull request merged**